### PR TITLE
Allow specifying multiple protocols on a single port

### PR DESCRIPTION
**Pull request by hyangtack**
Modifications:
- Change `protocol` field to `protocols` in `dogma.json`. Its type is also changed to an array.

### CODE REVIEW

File: server/src/main/java/com/linecorp/centraldogma/server/CentralDogmaConfig.java [hunk 1]
```diff
@@ -358,8 +360,8 @@ public ServerPort deserialize(JsonParser p, DeserializationContext ctx) throws I
                 return fail(ctx, root);
             }
 
-            final JsonNode proto = root.get("protocol");
-            if (proto == null || proto.getNodeType() != JsonNodeType.STRING) {
```
Comment (trustin) [inline server/src/main/java/com/linecorp/centraldogma/server/CentralDogmaConfig.java]: How about retaining this for backward compatibility?

Comment (hyangtack) [reply server/src/main/java/com/linecorp/centraldogma/server/CentralDogmaConfig.java]: Good point. Fixed!

File: site/src/sphinx/setup-configuration.rst [hunk 2]
```diff
@@ -67,9 +69,9 @@ Core properties
 
       - the port number. 36462 is preferred.
 
-  - ``protocol`` (string)
+  - ``protocols`` (set of string)
```
Comment (minwoox) [inline site/src/sphinx/setup-configuration.rst]: set of strings?

Comment (hyangtack) [reply site/src/sphinx/setup-configuration.rst]: Fixed. Thanks!

**Comment by hyangtack** [comment]
Thanks, reviewers!

**Pull request merged**